lib/package_utils.rb

```diff
@@ -2,12 +2,35 @@
 # Utility functions that take either a package object or a component of a package object as primary input.
 require 'json'
 require_relative 'const'
+require_relative 'convenience_functions'
 require_relative 'downloader'
 
 class PackageUtils
+  def self.reverse_dependency_lookup(dep_name, installed: false)
+    device_json       = ConvenienceFunctions.load_symbolized_json
+    pkgs_that_need_it = []
+
+    if installed
+      device_json[:installed_packages].each do |installed_pkg_info|
+        pkg_file      = File.join(CREW_PACKAGES_PATH, "#{installed_pkg_info[:name]}.rb")
+        installed_pkg = Package.load_package(pkg_file)
```

Comment:
```suggestion
        installed_pkg = Package.load_package(File.join(CREW_PACKAGES_PATH, "#{installed_pkg_info[:name]}.rb"))
```

just a visual preference, but it matches the pattern through the rest of the codebase